tools/src/test/java/com/nedap/archie/rmutil/PathableUtilTest.java

```diff
@@ -0,0 +1,75 @@
+package com.nedap.archie.rmutil;
+
+import com.nedap.archie.ArchieLanguageConfiguration;
+import com.nedap.archie.adlparser.ADLParser;
+import com.nedap.archie.adlparser.modelconstraints.RMConstraintImposer;
+import com.nedap.archie.aom.Archetype;
+import com.nedap.archie.query.RMObjectWithPath;
+import com.nedap.archie.query.RMPathQuery;
+import com.nedap.archie.query.RMQueryContext;
+import com.nedap.archie.rm.archetyped.Pathable;
+import com.nedap.archie.rm.composition.Composition;
+import com.nedap.archie.rm.datastructures.Cluster;
+import com.nedap.archie.rm.datastructures.ItemTree;
+import com.nedap.archie.rminfo.ArchieRMInfoLookup;
+import com.nedap.archie.rminfo.ModelInfoLookup;
+import com.nedap.archie.testutil.TestUtil;
+import com.nedap.archie.xml.JAXBUtil;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class PathableUtilTest {
```

Comment:
Maybe move this to the openehr-rm package, since the class you are testing is also there?

Reply:
This is currently not possible because this tests depends on TestUtil from the tools package.